informus/content/info_buttons_SPC.md

```diff
@@ -401,6 +401,40 @@ Means and process limits recalculate either after a sustained shift in the data
 
 ---
 
+**Page: SPC chart : Airway Plan)**
+
+## i button
+
+**What is this chart?**
+
+This chart is a Statistical Process Control (SPC) percentage chart. SPC charts are graphs that are commonly used in healthcare improvement for looking at a process over time and determining whether anything changes.  
+On this chart you will see a weekly percentage of patient who have had an airway plan documented in epic during their CCU admission, the average percentage over time (red mean line), and the data process limits (green hashed lines) which represent the boundary lines generated by the data, i.e. where we would expect most of the data to fall between. Note that process limits are not goal lines.  
+Outliers, shifts and trends suggest there is a non-random change to the data. Why are we seeing this change? Could it be improvement or deterioration to the process?  
+
+Means and process limits recalculate either after a sustained shift in the data OR every 6 months to reflect a more up-to-date process. 
+
+
+*Hyperlink -More about how to interpret SPC charts  (content for hyperlink see end of document)*
+ 
+The slider below the graph allows you to go forward and back in time. The maximum amount of retrospective data you can view is 112 weeks.
+
+
+## About Airway Management
+
+Airway management in critical care may be seen as a routine procedure for critically ill patients. Nevertheless, critical care patients are more prone to complications from airway management procedures, which may result in worse patient outcomes (such as death, brain damage and prolonged ICU stay).   
+
+All patients should have airway plan's documented in epic during admission and airway plans should be reviewed every 14 days. 
+
+
+## ? button
+
+**Eligibility**
+
+All patients across all units
+
+**Classification**
+
+A weekly percentage of all Critical Care inpatients who have had at least one airway plan documented in epic during there stay. Note that patients with admissions longer than one week will be recounted (in the percentage denominator) for each week of their stay, but once an airway plan is documented, it is valid for the rest of the admission. 
```

Comment:
Do you want the "More about how to interpret SPC charts" hyperlink under here on the `?` button content too? It's present in some but not all.